### DISCUSSION TITLE
No module named 'pyarrow.lib'

### DISCUSSION THREAD

**Discussion by thomasmore72** [datapane/datapane#201] [Help] [answered]
Hi,
This is a frustrating one for me, pyarrow is installed however non of the fixes have worked. has anyone else experienced this?

  File "C:\Python310\Lib\site-packages\pyarrow\__init__.py", line 63, in <module>
    import pyarrow.lib as _lib

ModuleNotFoundError: No module named 'pyarrow.lib'

**Comment by thomasmore72** [comment, marked as answer]
Update: somehow everything seems fine now so, I'm not sure what's changed.

**Reply by shahinrostami** [reply]
That's great news @thomasmore72! I was searching for similar issues, as I wasn't able to reproduce it. Please let us know if you encounter it again, and we can take a closer look!
